apps/frontend/src/components/ViewFormsList/ViewFormsList.tsx

```diff
@@ -27,6 +27,17 @@ export default function ViewFormsList() {
   const [sortOrder, setSortOrder] = useState<SortOrder>(SortOrder.ASC);
   const [searchTerm, setSearchTerm] = useState<string>('');
 
+  // the forms list will be filtered such that only forms whose hospital
+  // is in the hospitalsToFilter array and whose state is in the
+  // statesToFilter array will be shown; if an array is empty, then that
+  // type of filtering will not be performed
+
+  // should contain hospital abbreviations exactly as shown in the "hospital" column of the table (e.g., ["MASSGENHOSPITAL"])
+  const [hospitalsToFilter, setHospitalsToFilter] = useState<string[]>([]);
```

Comment:
Rather than using a string array as the type here, lets use a string of the hospital enum for the type (`enums/DropdownValues.ts`). That way it'll enforce that only valid hospital values can be added to this array.